Assets/Scripts/Player/Player Database/Responses/PlayerDataUpdateResponse.cs

```diff
@@ -0,0 +1,9 @@
+namespace Examen.Player.PlayerDatabase.Reponses
+{
+    public class PlayerDataUpdateResponse
+    {
+        public string success;
+        public string id;
+        public string exp;
```

Comment:
No it cannot be a struct because the API sends json back which can have some issues converting to structs. And Json also cannot change privates so they sadly need to be public.